app/scrapers/__init__.py

```diff
@@ -20,7 +20,7 @@
     'y': Yahoo(),
     'd': Duckduckgo(),
     'a': Ask(),
-    'yd': Yandex(),
+    'n': Yandex(),
```

Comment:
we should just use the full names instead of just one or two chars here

Reply:
Yes @AnshulMalik 
We have an issue where in we are going to change this initials setup and use the whole name.
This PR temporarily takes care of the yandex using yahoo.

Reply:
ok cool